### ISSUE TITLE
[BUG] Image `rancher/rke-tools:v0.1.105` is missing from `rancher-images.txt` in `v2.9.4-alpha3`

### ISSUE THREAD

**Issue by Josh-Diamond** [rancher/rancher#48026]
**Rancher Server Setup**
- Rancher version: `v2.9.4-alpha3
- Installation option (Docker install/Helm Chart): `Helm`

**Information about the Cluster**
- Kubernetes version: `v1.30.6-rancher1-1`
- Cluster Type (Local/Downstream): Downstream RKE1 w/ ECR private registry configured

**User Information**
- What is the role of the user logged in? `Admin`

**Describe the bug**
When provisioning a downstream AWS RKE1 node driver cluster w/ configured ECR private registry, rancher is unable to find `rancher/rke-tools:v0.1.105`, which results in the cluster failing to provision successfully

**To Reproduce**
1. Create an ECR private registry w/ images packaged in `v2.9.4-alpha3`
2. Provision a downstream RKE1 cluster using the ECR registry created in step 1
3. Reproduced

**Result**
Rancher fails to provision the downstream cluster due to the missing `rancher/rke-tools:v0.1.105`

**Expected Result**
The cluster provisions successfully

**Screenshots**
<img width="1275" alt="Screenshot 2024-11-12 at 3 14 42 PM" src="https://github.com/user-attachments/assets/ecfb7caa-3310-41a2-93c2-44f1ab6958ac">

**Additional context**
Only the following `rke-tools` images are packaged in `v2.9.4-alpha3`:
```
rancher/rke-tools:v0.1.100
rancher/rke-tools:v0.1.103
```

Furthermore, the hyperkube images packaged do not reflect the latest RKE versions associated w/ RKE `v1.6.4`, currently they are as follows:
```
rancher/hyperkube:v1.27.16-rancher1
rancher/hyperkube:v1.28.14-rancher1
rancher/hyperkube:v1.29.9-rancher1
rancher/hyperkube:v1.30.5-rancher1
```

But I would expect them to match the versions associated with RKE `v1.6.4`, shown in the screenshot below:
<img width="392" alt="Screenshot 2024-11-12 at 3 34 50 PM" src="https://github.com/user-attachments/assets/6bbf9086-d893-4d74-8ac3-71e1a498217e">

**Comment by kinarashah** [comment]
The `v2.9.4-alpha3` tag was created when the KDM branch was at `release-v2.9`. (https://github.com/rancher/rancher/blob/fb187f67a3a59699d373a94a6c92fa28023739b0/package/Dockerfile#L70) 

At that time, `release-v2.9` didn’t yet include the latest October patches—they were only present in the `dev-v2.9` branch. We just released KDM a few minutes ago so `release-v2.9` now has all the required versions. 

Once the PR to bump RKE is merged, we’ll need to tag a new alpha version to include October patches. If this doesn't work, I'll debug further but so far this looks like the root cause.

**Comment by kinarashah** [comment]
PR to bump RKE is merged - https://github.com/rancher/rancher/pull/48023. And `release/v2.9` is using KDM `release-v2.9`.
